less/home-page.less

```diff
@@ -58,18 +68,32 @@
       }
 
       .btn {
+        .btn-style();
       }
     }
 
     .walk-img {
-      width: 46%;
-      margin-left: 4%;
+      .walk-img-style();      
       
       img {
         width: 100%;
         height: auto;
         border-radius: 10px;
       }
     }
+    @media @phone {
+      flex-direction: column-reverse;
+      .walk-text {
```

Comment:
Good job using row reverse. You could have also set a order: -1; property to the image media query.